alacritty/src/event.rs

```diff
@@ -1546,6 +1506,29 @@ impl Processor {
 
                     info!("Initialisation complete");
                 },
+                // NOTE: try to request a redraw early on when we get a wakeup from the terminal
+                // to minimize the end input latency.
+                WinitEvent::UserEvent(Event {
+                    window_id: Some(window_id),
+                    payload: EventType::Terminal(TerminalEvent::Wakeup),
+                }) => {
+                    if let Some(window_context) = self.windows.get_mut(&window_id) {
+                        window_context.dirty = true;
+                        window_context.display.window.request_redraw();
+                    }
+                },
+                // NOTE: handle frame early on as well to not delay rendering much.
```

Comment:
```suggestion
                // NOTE: Handle frame early on as well to not delay rendering much.
```

If we always handle things early, what's the point in even mentioning it as something special? Should just state it in general then that this is something we want to do.